modules/tinymce/src/plugins/link/test/ts/browser/OpenLinkContextMenuTest.ts

```diff
@@ -92,4 +92,18 @@ describe('browser.tinymce.plugins.link.OpenLinkContextMenuTest', () => {
       'https://www.one.com/'
     ]);
   });
+
+  it('TINY-10391: Should open a new tab when triggering open link on an image with embedded link', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p><a href="https://www.exampleimage.com"><img src="http://www.example.image/image"></a></p>');
+    // Open link context menu on the link
+    await TinyUiActions.pTriggerContextMenu(editor, 'a[href="https://www.exampleimage.com"]', '.tox-silver-sink [role="menuitem"]');
```

Comment:
should you wait for editor to focus on the first item of the context menu before keydown?
`pTriggerContextMenu` only wait for the menu to show up, doesn't check for focus

Reply:
Should not matter because the context menu handles the keydowns